server/controllers/games.js

```diff
@@ -0,0 +1,73 @@
+const Game = require('../models').Game;
+
+
+module.exports= {
+  create(req, res) {
```

Comment:
We should make sure that we add internal documentation to all our functions.

For controller actions, it would be important to also include what urlParams, and bodyParams are used in the action